.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: go
+
+go:
+  - 1.10.x
+
+before_install:                                                                 
+  - curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh  
```

Comment:
Is this step really needed given the `godep` support in Travis: https://docs.travis-ci.com/user/languages/go/#godep-support?

If it's needed, add a comment explaining what this does. (I initially thought that this installed Golang).

Reply:
Travis does not support `golang\dep` out of the box ( I assume they should fix it as more and more people are migrating to `golang\dep`). I'll add a comment though